lib/view_model/access_control/tree.rb

```diff
@@ -192,8 +192,6 @@ def root?
       def inspect_checks
         checks = super
         if root?
-          checks << 'no root checks'
```

Comment:
this was just completely backwards - on a root note it output a lie, and on a non-root node it fruitlessly went through all the checkers that would necessarily be empty